### PR TITLE
feat-176-client-responsive-page

### PR DESCRIPTION
**Pull request by zubairaltaf10**

### CODE REVIEW

File: components/unzipped/TrustedBy/TrustedBy.js
Comment (Alien4Hire) [inline components/unzipped/TrustedBy/TrustedBy.js:1]: Seeing issue on desktop screen of this
image

Comment (Alien4Hire) [inline components/unzipped/TrustedBy/TrustedBy.js:1]: Seeing some issues on footer and other content being shifted. Please check at 450px and 325px.

image

Comment (Alien4Hire) [inline components/unzipped/TrustedBy/TrustedBy.js:1]: Button is not correct height in pixel

image

Comment (Alien4Hire) [inline components/unzipped/TrustedBy/TrustedBy.js:1]: Read more icon should turn to point an arrow upwards when open to show all content

image

Comment (zubairaltaf10) [reply components/unzipped/TrustedBy/TrustedBy.js:1]: Figma design has 188px width and 38px height it is correct in height. If you want to me to give it some more height i can give it

**Pull request merged**